src/classifier.jl

```diff
@@ -101,6 +101,9 @@ function MLJBase.update(model::NeuralNetworkClassifier, verbosity::Int, old_fitr
         chain = old_chain
         epochs = model.n - old_model.n
     else
+        n = MLJBase.schema(X).names |> length
```

Comment:
this `n` is the dimensionality? if so would you mind using either  `p` or `d` for consistency + avoid confusion with number of samples? maybe just use an explicit  var name. 

Same for `m` below which  could be `n_classes_seen`

Reply:
So, `d`, I take it. Better would be `n_features`.